acousticbrainz/testing.py

```diff
@@ -21,16 +22,17 @@ def tearDown(self):
         self.truncate_all()
 
     def truncate_all(self):
-        conn = psycopg2.connect(self.app.config["PG_CONNECT"])
-        cur = conn.cursor()
-        cur.execute("TRUNCATE highlevel_json CASCADE;")
-        cur.execute("TRUNCATE highlevel CASCADE;")
-        cur.execute("TRUNCATE lowlevel CASCADE;")
-        cur.execute("TRUNCATE statistics CASCADE;")
-        cur.execute("TRUNCATE incremental_dumps CASCADE;")
-        conn.commit()
-        cur.close()
-        conn.close()
+        connection = psycopg2.connect(current_app.config['PG_CONNECT'])
```

Comment:
That's it -- not meant to. But if it were to happen accidentally (or habitually) then we lose our production DB. :(